qa/L0_model_update/instance_update_test.py

```diff
@@ -457,15 +467,115 @@ def test_instance_resource_decrease(self):
                 # explicit limit of 10 is set.
                 self.assertNotIn("Resource: R1\t Count: 3", f.read())
 
-    # Test for instance update on direct sequence scheduling
-    @unittest.skip("Sequence will not continue after update [FIXME: DLIS-4820]")
-    def test_instance_update_on_direct_sequence_scheduling(self):
-        pass
-
-    # Test for instance update on oldest sequence scheduling
-    @unittest.skip("Sequence will not continue after update [FIXME: DLIS-4820]")
-    def test_instance_update_on_oldest_sequence_scheduling(self):
-        pass
+    # Test instance update for direct and oldest scheduler
```

Comment:
This comment doesn't help me understand what the test setup is and what is the expected result from performing the test. To understand the test, I still need to look at the detail of the test.

And you are testing 4 test case ([direct, oldest] X [basic, ongoing]) in this test, please separate them into individual tests unless the current form is the minimal unit of the testing. Otherwise, it requires additional effort to distinguish which case is failing when this "collected" test fails.

Reply:
I think we do not need to break it into 4 cases, because the test will show exactly which line that failed, which will give us a very good hint onto what went wrong. The only issue I had in the past is not knowing the line that failed represent whether the direct batcher or oldest batcher, so I separated it into 2 cases - direct and oldest.

Update commits: [Separate direct and oldest to different test cases](https://github.com/triton-inference-server/server/pull/5831/commits/c9d7b5f3f3059add7cea34a997a45bac27c84525) and [Separate nested tests in a loop into multiple test cases](https://github.com/triton-inference-server/server/pull/5831/commits/dcb55f0c047a0bc7c88f1fde7693bcfc3a01b89b)

Reply:
Further split the cases for cleanness: [Refactor scheduler update test](https://github.com/triton-inference-server/server/pull/5831/commits/0268918bfbf613fdba943ea5dde34d3bd62a5534)